Formula/o/openstackclient.rb

```diff
@@ -20,33 +20,35 @@ class Openstackclient < Formula
   depends_on "rust" => :build # for rpds-py
   depends_on "cffi"
   depends_on "pycparser"
+  depends_on "python-attrs"
```

Comment:
I know some of these were already here, but let's not add any new `depends_on` for python packages. The `resource` blocks created with `brew update-python-resources` are fine